requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyDictionary~=2.0.1
```

Comment:
We currently have a different requirements file called (quite confusingly) [dev-requirements.txt](https://github.com/Yagueteiro/code-jam-2021/blob/main/dev-requirements.txt)

Reply:
Usually you try to keep dev and user requirements separate. Are you sure?

Reply:
Oh ok, I had never seen this file and assumed wrong lol